CONTRIBUTING.md

```diff
@@ -37,14 +35,43 @@ The channel name is
 
 <pre> #project-cre</pre>
 
-
 ## How Can I Contribute?
 
-The "Issues" page lists a number of features we would like to implement, we have tagged the ones we believe are easy to pick up with the tag `good first issue` and/or `beginner`. Alternatively you can contribute content by adding a mapping or request features or mappings by opening an Issue.
+The "Issues" page lists a number of features we would like to implement, we have tagged the ones we believe are easy to pick up with the tag `good first issue` and/or `beginner`. Alternatively you can contribute content (see below) or request features or mappings by opening an Issue.
+
+
+### How can I contribute a mapping or change the catalog of CREs?
+
+Adding a mapping to OpenCRE for a new standard X means that each section in X is assigned to the corresponding ‘Common Requirement’ (or CRE number) at opencre.org.
+For example, the section 613-Insufficien Session expiration in the CWE standard is mapped to CRE 065-782 Ensure session timeout (soft/hard).
+The result is that when you go to the overview page of that requirement, users will see a link to CWE 613: https://www.opencre.org/cre/065-782
+
+How to:
+1. Get the OpenCRE standard mapping template spreadsheet
+2. For every section in the standard, find the corresponding Common Requirement (CRE number) at OpenCRE and enter in that row the details of that section in the right columns:  name, id, and hyperlink 
+3. In case you identify opportunities to add Common Requirements: add those to the spreadsheet
+4. Send the mapping template to rob.vanderveer@owasp.org. You can also use that mail address for any questions.
```

Comment:
For visibility, can we also get contributors to open a ticket? It's ok if not, just tickets show traffic and interest